torchinductor/config.py

```diff
@@ -49,6 +49,8 @@
 # replace small reductions with pointwise, disable with `= 1`
 unroll_reductions_threshold = 8
 
+comment_origin = True
```

Comment:
We should turn this off by default because it could increase compile times by preventing cache hits at the triton level.